Stacking tetromino sample

AlexJerabek opened the pull request:
This is a fun *meditation exercise* about stacking *tetromino shapes* and *counting* the rows you've cleared. Part of the PR to make sure alternatives to those italicized parts aren't used.

kbrandl commented:
@AlexJerabek did you run `npm start` (to verify that everything builds successfully, and that the Excel view and playlist are updated as necessary)?

AlexJerabek commented:
Thanks @kbrandl. I knew I was missing something... Moved the sample to preview, since it uses the beta libraries. Also responded to offline feedbac from @jakobpn about control margins and spelling tetr**o**mino correctly.

AlexJerabek commented:
@kbrandl Fixed the background image, as per your meeting comment. If you have any more usage questions, please swing by, I'm happy to answer them.

kbrandl commented on samples/excel/85-preview-apis/tetrominos.yaml:
```diff
@@ -0,0 +1,796 @@
+                    usageElem.classList.add("usage");
+                    usageElem.innerHTML =
+                        "<table>" +
+                        "<tr><th>Cursor Keys</th><td>Steer</td></tr>" +
```
Some suggestions:

- Add a heading before this list of keys/commands that explains what the list is...e.g., maybe **How to play** or something along those lines...and consider adding some text after the heading, something along the lines of "Use your keyboard to play the game:"
- Line 1: Change **Cursor Keys** to **Left arrow key** and **Right arrow key** (assuming that's what you mean by "Cursor Keys") and split those into two separate lines -- one with description **Move shape left** and the other with description **Move shape right**
- Line 2: Split the **'a'/'d'** line into two separate lines -- one with title **'a' key** and the other with title **'d' key** -- and with descriptions of **Rotate shape clockwise** and **Rotate shape counterclockwise** as appropriate
- Line 3: Update title to **Space key** and description to **Let shape fall**
- Line 4: Update title to **Enter key**
- Line 5: Update title to **'R' key**

AlexJerabek commented:
Thanks again @kbrandl. I fixed the session pane disappearing if you're in cell-edit mode. I also added a header for the "next shape preview". Finally, I improved the control description. I looked up some examples (that are just coincidentally games) of how they display and describe keyboard controls. [Castle Crashers](https://www.bing.com/images/search?view=detailV2&ccid=NfkTF%2bX3&id=9EB18246772EDC5550B6FD863A0937CA5A080710&thid=OIP.sQhqpSaI_9XNiId2AKeZrgHaEK&mediaurl=https%3a%2f%2fi.stack.imgur.com%2fnWGs5l.jpg&exph=360&expw=640&q=keyboard+game+contol+screens&simid=607994765703974887&selectedIndex=31&ajaxhist=0) shows the action, followed by the control key, without the without "key".

AlexJerabek commented:
Also, I was able to get that error once. I wonder if that's coming from Script Lab itself. @Zlatkovsky, do you have any insight to the listed error?

AlexJerabek commented on samples/excel/85-preview-apis/tetrominos.yaml:
```diff
@@ -0,0 +1,783 @@
+                        "<tr><th>Move shape right</th><td>Right arrow</td></tr>" +
+                        "<tr><th>Move shape down</th><td>Down arrow</td></tr>" +
+                        "<tr><th>Rotate counterclockwise</th><td>A</td></tr>" +
+                        "<tr><th>Rotate clockwise</th><td>D</td></tr>" +
```
@kbrandl There's a MSFT style guide article saying key references should be capitalized. Here's a [public example](https://support.microsoft.com/en-us/help/17073/windows-using-keyboard). My guess is that's because the physical label is always a capital letter.

kbrandl replied:
Per our in-person chat just now, let's keep the capital letters :)

AlexJerabek commented:
Updated with feedback from @Zlatkovsky. @Zlatkovsky, can I get your approval on this this?
